src/ipv4.c

```diff
@@ -885,13 +885,23 @@ int ipv4_add_split_vpn_route(struct tunnel *tunnel, char *dest, char *mask,
 
 static int ipv4_set_split_routes(struct tunnel *tunnel)
 {
-	int i;
+	int i, j;
```

Comment:
Perhaps move this declaration of `j` inside the `i` loop.

Reply:
done.